stdlib/LinearAlgebra/src/eigen.jl

```diff
@@ -47,28 +44,45 @@ julia> vals == F.values && vecs == F.vectors
 true
 ```
 """
-struct Eigen{T,V,S<:AbstractMatrix,U<:AbstractVector} <: Factorization{T}
+struct Eigen{T,V,S<:AbstractMatrix,U<:AbstractVector,R<:AbstractVector} <: Factorization{T}
     values::U
     vectors::S
-    Eigen{T,V,S,U}(values::AbstractVector{V}, vectors::AbstractMatrix{T}) where {T,V,S,U} =
-        new(values, vectors)
+    vectorsl::S
+    eerrbd::R
+    verrbd::R
+    Eigen{T,V,S,U,R}(values::U, vectors::S, vectorsl::S,
+                     eerrbd::R, verrbd::R,
+                    ) where {T,V,S<:AbstractMatrix{T},U<:AbstractVector{V},R} =
+        new(values, vectors, vectorsl, eerrbd, verrbd)
+end
+
+function Eigen(values::U,
+               vectors::S,
+               vectorsl::S=vectors,
+               eerrbd::R=emptyvec(T),
+               verrbd::R=emptyvec(T),
+              ) where {T,V,R,S<:AbstractMatrix{T},U<:AbstractVector{V}}
+
+    Eigen{T,V,S,U,R}(values, vectors, vectorsl, eerrbd, verrbd)
 end
-Eigen(values::AbstractVector{V}, vectors::AbstractMatrix{T}) where {T,V} =
-    Eigen{T,V,typeof(vectors),typeof(values)}(values, vectors)
 
 # Generalized eigenvalue problem.
 """
     GeneralizedEigen <: Factorization
 
-Matrix factorization type of the generalized eigenvalue/spectral decomposition of
+Matrix factorization type of the generalized eigenvalue/apply decomposition of
```

Comment:
?

Reply:
That was a edit error, should be unchanged.